src/redux/slices/feedSlice.ts

```diff
@@ -3,31 +3,140 @@ import { FeedItem } from "../../utilities/types";
 
 interface feedState {
   feed: FeedItem[];
+  newFeed: FeedItem[];
+  replies: FeedItem[];
+  currentBlock: number | null;
 }
```

Comment:
Would like some feedback on the way I'm storing/managing my feed data. I'm not 100% happy with the way it works. I feel it could be more efficient and perhaps a bit cleaner.